packages/base/src/3dview/mainview.tsx

```diff
@@ -1000,6 +1000,13 @@ export class MainView extends React.Component<IProps, IStates> {
         selectedMesh.remove(boundingBox);
       }
```

Comment:
I believe we also don't need to remove `boundingBox` from `selectedMesh` too, now that you don't add it to the mesh anymore :)

Reply:
Oops! I should've thought that 😅